gap/profiling.gi

```diff
@@ -787,7 +787,7 @@ function(data, outfile, jobid, pathtoremove)
 
     IO_Write(outstream, "{\n");
     IO_Write(outstream, Concatenation("\"service_job_id\": \"", jobid, "\",\n"));
-    IO_Write(outstream, "\"service_name\": \"travis-ci\",\n");
+    IO_Write(outstream, Concatenation("\"service_name\": \"", service_name, "\",\n");
```

Comment:
There are additional key/value pairs we may need to set. So I suggest that if we break things anyway, we rather change this API as follows: drop the `jobid` and `service_name` arguments to this function, instead pass in an `opt` record. Then write each key/value pair of that opt record into the file. To keep things simple, let's restrict the "values" to strings (and perhaps ints).

Then we can also set the `service_pull_request` and `service_branch` keys and more, as "documented" (haha) here: https://github.com/lemurheavy/coveralls-ruby/blob/master/lib/coveralls/configuration.rb

Reply:
I guess we could also allow `opt` to be optional, and in that case, try to "guess" the CI data from the environment, as done in the file I just linked to.